index.html

```diff
@@ -66,8 +69,8 @@
      </section>
     </footer>
 
-    <script src='https://code.jquery.com/jquery-2.2.1.min.js'></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.4/handlebars.js"></script>
+    <script src='http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js'></script>
+    <script src='https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.4/handlebars.js'></script>
     <script src='scripts/projectView.js'></script>
     <script src='scripts/app.js'></script>
     <script>
```

Comment:
Good catch.  Yes be consistent.  Also if you are going to be moving on to the Full Stack JavaScript.  You can use one type of quotes on the client side and the other on the server side.